src/CityOnPlanet.cpp

```diff
@@ -266,23 +381,110 @@ CityOnPlanet::CityOnPlanet(Planet *planet, SpaceStation *station, const Uint32 s
 	orientcalc[2] = m * matrix4x4d::RotateYMatrix(M_PI * 0.5 * 2);
 	orientcalc[3] = m * matrix4x4d::RotateYMatrix(M_PI * 0.5 * 3);
 
-	const double maxdist = pow(gmid + 0.333, 2);
-	for (int x = 0; x <= gsize; x++) {
-		const double distx = pow((x - gmid), 2);
-		for (int z = 0; z <= gsize; z++) {
-			if (cellgrid[x][z] > 0) {
-				// This cell has been allocated for something already
+#ifdef DEBUG_OUTPUT
+	Output("CityOnPlanet: Station '%s' X %i .. %i, Z %i .. %i, pop %.0f cityradius %i\n",
+		stationmodelname, station_x1, station_x2, station_z1, station_z2, pop, cityradius);
+
+	int skipped = 0, placed = 0, emptycells = 0, occupiedcells = 0, loopcells = 0, rarityrolls = 0;
```

Comment:
This can stay in even in non-debug, that'll help to get rid of the `#ifdef DEBUG_OUTPUT` spam later on in the code